src/content/docs/node/introduction.mdx

```diff
@@ -1,3 +1,502 @@
+| **Development speed** | Fast with JavaScript across stack   | Fast, easy to learn                | Fast, easy to deploy              |
+
+## 1. Basic Installation Steps for Node.js
+
+    <Steps>
+
+    1. **Download Node.js**:
+
+       - Visit the official Node.js website: [Node.js Official Website](https://nodejs.org).
+       - Download the recommended version for your operating system (LTS is recommended for most users).
+
+    2. **Install Node.js**:
+
+       - Run the installer you downloaded.
+       - Follow the installation prompts, which usually involves agreeing to the license agreement and choosing the installation path.
+
+    3. **Verify Installation**:
+
+       - Open a terminal or command prompt and run the following commands to check if Node.js and npm (Node Package Manager) are installed correctly:
+
+         ```bash
+         node -v
+         npm -v
+         ```
+
+    </Steps>
+
+## 2. Code Editor:
+
+        **Visual Studio Code (VS Code)**:
+            - Lightweight and feature-rich.
+            - Install it from the [official website](https://code.visualstudio.com/).
+
+        **Recommended extensions**:
+            - **Node.js Extension Pack**: Adds debugging, IntelliSense, and tools for Node.js.
+            - **ESLint**: Helps in identifying and fixing coding errors.
+            - **Prettier**: Automatically formats your code.
+
+## 3. Version Control: Git
+
+    Version control helps you manage code changes effectively.
+        - Install Git from the official website.
+        - Set up Git in your project directory:
+         ```bash
+        git init
+        git config --global user.name "Your Name"
+        git config --global user.email "your.email@example.com"
+        ```
+
+## 4. Project Setup
+
+    Start by creating a new Node.js project:
+
+    <Steps>
+
+    1. **Download Node.js**:
+
+        ```bash
+        mkdir my-node-project
+        cd my-node-project
+        ```
+    2. **Initialize the project**:
+
+        ```bash
+        npm init -y
+        ```
+
+    </Steps>
+
+## 5. Useful NPM Packages
+
+    Some essential packages to install right away:
+        - Express: A minimalist web framework to install use **npm install express**.
+        - Nodemon: Automatically restarts the server when you make changes use **npm install --save-dev nodemon**.
+        - dotenv: Manages environment variables **npm install dotenv**.
+
+## 6. Folder Structure
+
+    A good folder structure keeps your code organized.
+
+        <FileTree>
+
+        - /my-node-project
+            - /config
+              - /db.js
+            - /controllers
+              - /userController.js
+            - /models
+              - /userModel.js
+            - /routes
+              - /route.js
+          - .env
+          - index.js
+          - package.json
+
+        </FileTree>
+
+## Creating Your First Node.js Application: A Step-by-Step Tutorial
```

Comment:
In this **"Creating Your First Node.js Application: A Step-by-Step Tutorial"** please remove the **": A Step-by-Step Tutorial"**